### PR TITLE
Group16 to xyzOrthographicProjection

### PR DESCRIPTION
**Pull request by KyrenePineda**
Tests are not great. I will address any problems tomorrow as soon as I can.

### CODE REVIEW

File: xyzOrthographicProjection.c [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#include xyzOrthographicProjection.h
+
```
Comment (andrewlinington) [inline xyzOrthographicProjection.c:2]: all of these files should be in your corresponding folder renamed to your functionality

File: xyzOrthographicProjectionTests.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#ifndef XYZ_ORTHOGRAPHIC_PROJECTION_TESTS_H
+#define XYZ_ORTHOGRAPHIC_PROJECTION_TESTS_H
+
+#include "../main.h"
```
Comment (andrewlinington) [inline xyzOrthographicProjectionTests.h:4]: makefile is not updated please update any changes found there

**Review by andrewlinington — CHANGES_REQUESTED** [review]
fix folder issue, update the makefile, and please run your code using make then ./main with valgrind as found in the readme as that will speed up the process for approval

**Review by andrewlinington — CHANGES_REQUESTED** [review]
image
 review the naming of some of your functions and consider naming them something specific to your group as i have noticed a few groups use that same naming convention.

**Review by andrewlinington — APPROVED** [review]
Tests still need to be fixed but im merging the actual code

**Review by cc0407 — APPROVED** [review]

**Pull request merged**